articles/quickstart/spa/angular2/02-user-profile.md

```diff
@@ -17,7 +17,7 @@ budicon: 292
 
 ## Request the Profile Scope
 
-The user's `access_token` requires a `scope` of `openid profile` to successfully retrieve their information. In the `WebAuth` instance, specify that you would like to ask for these scopes.
+To retrieve user information, request a scope of `openid profile` in the `WebAuth` instance. 
```

Comment:
Let's use auth0.WebAuth. In general you create a single one, if you see an example where there are more than one, let me know.